### ISSUE TITLE
WA: drop-down menu markup

### ISSUE THREAD

**Issue by alisonjo315** [CU-CommunityApps/cwd_base#42]
⭐ _**From Alison: Please update this issue description however y'all see fit, I'm just getting it started -- reword/replace details, whatever.**_

----
### Background
During WA review of [CRS Policy Office](https://test-crs-policy-office.pantheonsite.io/), an issue was raised with the drop-down menu markup (main menu).  Menu items had `aria-expanded="false"` and didn't change to `true` when drop-downs were in fact expanded.<br>
_(NOTE: The policy site isn't like that anymore, I removed the attribute so we could launch.  [Here's an old multidev that still has the `aria-expanded` issue](https://pol-mig-stg-crs-policy-office.pantheonsite.io/), if you want to see it -- I can't promise the multidev will be there when you look...)_

CRS Policy Office was built with an older CD Demo theme (cwd_ssit), like the other CRS sites.
Other CRS sites have `aria-expanded` but it works properly -- for example, [CRS Privacy Office](https://privacy.cornell.edu/). 
On newer Drupal 8 sites (and on CD Demo), `aria-haspopup` is used, but `aria-expanded` is not used at all.

### Questions:
* Is the markup on CD Demo good, or should it be modified?
* If it should be modified, what should it be?
  * If it were modified, we'd do it in cwd_framework and cwd_base.
    * [menu--main.html.twig](https://github.com/CU-CommunityApps/cwd_base/blob/main/templates/menu--main.html.twig)
    * maybe [cwd_utilities.js](https://github.com/CU-CommunityApps/cwd_base/blob/main/js/cwd_utilities.js)
    * wherever the equivalent code exists in [cwd_framework](https://github.com/CU-CommunityApps/cwd_framework)
* I haven't mentioned non-Drupal-8 sites at all, so uh, someone should look at WordPress or whatever.

### Links:
* CD Demo: https://dev-cd-demo.pantheonsite.io/
* Framework demo site: https://iws-preview.hosting.cornell.edu/ama39/cssf/
* [April 2020 update to framework repo](https://github.com/CU-CommunityApps/cwd_framework/commit/852cad01fd37b8a3920af9b7dbaaf0d377c430cd#diff-da93429357500c6c481ac8bb8d57c68755c5f799c3dd57fdee571bf45207fbc3), with keyboard nav improvements to main menu.

**Comment by alisonjo315** [comment]
BLERG.  This issue should be on cwd_framework, so, I'm closing it here and adding it over there...

https://github.com/CU-CommunityApps/cwd_framework/issues/9
